packages/block-library/src/quote/edit.js

```diff
@@ -132,7 +132,9 @@ export default function QuoteEdit( {
 								createBlock( getDefaultBlockName() )
 							)
 						}
-						{ ...( ! isWebPlatform ? { textAlign: align } : {} ) }
+						{ ...( ! isWebPlatform
+							? { textAlign: attributes.textAlign }
+							: {} ) }
```

Comment:
```suggestion
						{ ...( ! isWebPlatform ? { textAlign } : {} ) }
```

We can make this more concise given `textAlign` has already been destructured from attributes earlier.